public/vocables/js/search/searchUtil.js

```
@@ -37,7 +37,12 @@ var SearchUtil = (function () {
                                 if (err) return callbackSeries(err);
 
                                 indexes = [];
-                                if (toSources && !Array.isArray(toSources)) toSources = [toSources];
+                                if (toSources) {
+                                    if (!Array.isArray(toSources)) toSources = [toSources];
```

if toSources has to be an array, let's raise an exception when it is not.